aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -2773,6 +2773,22 @@ s3-control:
   tags:
     product_categories: {'Storage'}
   version: s3control-2018-08-20
+s3-directory-buckets:
+  bundle: s3
+  api_ref: AmazonS3/latest/API/Welcome.html
+  blurb: are a high-performance, single-zone Amazon S3 storage class that is purpose-built to deliver consistent, single-digit millisecond data access for your most latency-sensitive applications.
```

Comment:
are designed to store data within a single AWS Zone. Directory buckets organize data hierarchically into directories, providing a structure similar to a file system.